addons/l10n_es_edi_sii/models/account_edi_format.py

```diff
@@ -46,6 +46,7 @@ def get_connection(self, url, proxies=None):
         context = conn.conn_kw['ssl_context']
 
         def patched_load_cert_chain(l10n_es_odoo_certificate, keyfile=None, password=None):
+            l10n_es_odoo_certificate.sudo().password
```

Comment:
I don't understand why you do it here, could you explain, please?
When I tried, just adding two sudos where the password is accessed in _decode_certificate did the trick and kept it as specific as possible.
And it didn't need any changes in the ir.model.access too.

Reply:
I discussed this with WAN (because I don't know much about ormcache :speak_no_evil: : ) 
Because of the ormcache, he said that we should add the sudo() when calling the method: l10n_es_odoo_certificate.sudo()._decode_certificate()
His opinion is that we shouldn't give access to the certificate to other groups. It is part of the configuration and other groups have no reason to have direct access to it.

> Concerning the new access in ir.model.access I don't see how we could do without it because non-admin users need those rights to access the certificate and send invoices to the SII

At least from my quick test yesterday, I could click the send now button and it would do everything without issue, without the need to add the access rights (but maybe other flows use it?). What should be done is add sudo at other places if there is any that needs the certificate, I think.

Reply:
Should be good now :ok_hand: